citrination_client/search/client.py

```diff
@@ -123,8 +123,8 @@ def _search_internal(self, returning_query, result_class):
             failure_message = "Error while making dataset search request"
 
         response_json = self._get_success_json(self._post(
-                    route, data=json.dumps(returning_query, cls=QueryEncoder),
-                failure_message=failure_message))
+                    route, data = json.dumps(returning_query, cls = QueryEncoder),
```

Comment:
I *believe* [PEP8](https://www.python.org/dev/peps/pep-0008/) (the de facto Python style guide) says that keyword args like this do not have spaces around the equals sign. Personally, I don't actually care that much. But just FYI.

Reply:
Thanks, I will switch back to PyCharm